### PR TITLE
Moved python cheatsheet under cheatsheet file

### PR DESCRIPTION
**Pull request by ajs1220**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Added the python cheatsheet under the python cheatsheet file.
### How do we know it's really free?
This is free because it is freely available on the internet
### For book lists, is it a book? For course lists, is it a course? etc.
It is a document.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -328,6 +328,7 @@
 * [Comprehensive Python Cheatsheet](https://gto76.github.io/python-cheatsheet) - Jure Šorn (HTML)
 * [Learn Python in Y minutes](https://learnxinyminutes.com/docs/python/) - LearnXinYMinutes (HTML)
 * [Official Matplotlib cheat sheets](https://github.com/matplotlib/cheatsheets) - Matplotlib.org (LaTeX, PDF)
+* [Python Basics Cheat Sheet](https://www.utc.fr/~jlaforet/Suppl/python-cheatsheets.pdf) - Data Camp (PDF)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:331]:
```suggestion
* [Python For Data Science Cheat Sheet](https://www.utc.fr/~jlaforet/Suppl/python-cheatsheets.pdf) - Data Camp (PDF)
```
Please provide the names provided by the resource

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Review by shrn01 — APPROVED** [review]

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -328,8 +328,9 @@
 * [Comprehensive Python Cheatsheet](https://gto76.github.io/python-cheatsheet) - Jure Šorn (HTML)
 * [Learn Python in Y minutes](https://learnxinyminutes.com/docs/python/) - LearnXinYMinutes (HTML)
 * [Official Matplotlib cheat sheets](https://github.com/matplotlib/cheatsheets) - Matplotlib.org (LaTeX, PDF)
-* [Python Cheat Sheet](https://websitesetup.org/python-cheat-sheet/) - WebsiteSetup (HTML, PDF, PNG)
+* [Python For Data Science Cheat Sheet](https://www.utc.fr/~jlaforet/Suppl/python-cheatsheets.pdf) - Data Camp (PDF)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:331]: Can you provide the official link from data camp

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: more/free-programming-cheatsheets.md (hunk 2 above)
Comment (shrn01) [inline more/free-programming-cheatsheets.md:331]: Looks like multiple cheatsheets were compiled into one here. Data Camp website has each page as a seperate cheatsheet. What can we do about this?

https://www.datacamp.com/cheat-sheet/category/python

Comment (eshellman) [reply more/free-programming-cheatsheets.md:331]:
```suggestion
* [Python For Data Science Cheat Sheet](https://www.datacamp.com/cheat-sheet/python-for-data-science-a-cheat-sheet-for-beginners) - Data Camp (PDF)
```

**Pull request merged**